WARNINGS.md

```diff
@@ -334,11 +334,12 @@ just use a list of strings.
   * Automatic fix: no
 
 Public functions should have docstrings describing functions and their signatures.
-A docstring is a string statement which should be the first statement of the file
-(it may follow comment lines). Docstrings are expected to be formatted in the
-following way:
+A docstring is a string literal (not a comment) which should be the first statement
+of a function (it may follow comment lines). Function docstrings are expected to be
+formatted in the following way:
 
-    """One-line summary: must be followed and may be preceded by a blank line.
+    """
+    One-line summary: must be followed and may be preceded by a blank line.
```

Comment:
Convention is to write the summary on the same line as `"""`.

Reply:
Fixed.